### PR TITLE
Add allocation test that parses all WebPKI roots

### PR DESCRIPTION
**Pull request by dnadoba**
### Motivation
We want to start improving the allocation count that we do during decoding. 

### Modifications
- add 5 roots certificates from the WebPKI through `WebPKI.roots`
- add an allocation test that parses all `WebPKI.roots` and gets the number of extensions

### Result
We track the number of allocation we do during certificate parsing.

### CODE REVIEW

File: IntegrationTests/tests_01_allocation_counters/test_01_resources/test_parse_webpki_roots.swift [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            do {
+                var totalExtensionCount = 0
+                for _ in 0..<1000 {
+                    for derEncodedCA in derEncodedCAs {
+                        totalExtensionCount += try Certificate(derEncoded: derEncodedCA).extensions.count
+                    }
+                }
+                return totalExtensionCount
+            } catch {
+                fatalError("\(error)")
```
Comment (Lukasa) [inline IntegrationTests/tests_01_allocation_counters/test_01_resources/test_parse_webpki_roots.swift:32]: Replace with `try!`

File: IntegrationTests/tests_01_allocation_counters/test_01_resources/test_parse_webpki_roots.swift [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            }
+        }
+    } catch {
+        fatalError("\(error)")
```
Comment (Lukasa) [inline IntegrationTests/tests_01_allocation_counters/test_01_resources/test_parse_webpki_roots.swift:36]: Replace with `try!`.

Comment (dnadoba) [reply IntegrationTests/tests_01_allocation_counters/test_01_resources/test_parse_webpki_roots.swift]: ahhh... of course... stupid me 🙃

**Review by Lukasa — APPROVED** [review]
Nice one!

**Pull request merged**